### ISSUE TITLE
Element not interactable errors when trying to click the down-arrow bounce button

### ISSUE THREAD

**Issue by kchaya1** [dvalbuena1/visa_rescheduler_aws#20]
The way the code works rn relies on the code clicking this button: 

<img width="1650" alt="Screenshot 2023-11-25 at 12 12 04 AM" src="https://github.com/dvalbuena1/visa_rescheduler_aws/assets/134338884/6c5e248e-d02f-47ae-b694-6ab607986ffc">

right here:
https://github.com/dvalbuena1/visa_rescheduler_aws/blob/main/visa.py#L112

However, I got some errors saying that this element is not interactable. I modified my lambda to take a screenshot of the current webpage and store it in s3 whenever this happens. This is what that screenshot looks like:
![5843e32d-37dc-4538-93a6-3fc0400fe8d5](https://github.com/dvalbuena1/visa_rescheduler_aws/assets/134338884/3295c12a-e066-4c5a-b84a-4231b18e1001)

SO basically, there are times when that down-arrow bounce doesn't appear on the website.

I also logged out all the CSS elements on the page whenever this happens. For the down arrow bounce, this is what my logs say:

`INFO:log_css_elements_upload_screenshot_s3 - Element tag: a, class: down-arrow bounce, is_visible:False, is_enabled:True, interactable: False`

So basically it is "enabled" but not "visible", and thus it cant be interacted with. Anyone else see this issue? So sometimes, the down-arrow bounce is not there. I think we will need some special logic to check if its not interactable, proceed with the rest of the steps anyway.

**Comment by dvalbuena1** [comment]
Sorry for the late reply,

What you could do is to add something like 
```py
try:
     a = self.driver.find_element(By.XPATH, '//a[@class="down-arrow bounce"]')
     a.click()
     tm.sleep(STEP_TIME)
except NoSuchElementException:
     # No arrow
```

In this way, if there is no arrow you will not get any error, it is a way just to skip that step

**Comment by Jleotorres1** [comment]
@dvalbuena1 el condigo esta en funcionamiento? he agregado al codigo principal la restricccion que encuentre la cita del CAS un dia antes que de la embajada, esto es necesario? o el codigo en si lo realiza? has probado ultimamente el codigo a ver si puedes reprogramar citas?
